tidy3d/components/mode.py

```diff
@@ -91,6 +91,15 @@ class ModeSpec(Tidy3dBaseModel):
         "yz plane, the ``bend_axis`` is always 1 (the global z axis).",
     )
 
+    track: Literal["first", "last", "center"] = pd.Field(
+        None,
```

Comment:
another comment, I'm not sure about calling the field "track", maybe it is not specific enough. Maybe `track_freq`? And also, I feel instead of eg. "first" maybe "lowest"? since we're talking about frequencies?

Reply:
I suggest setting `"central"` as default but still allowing `None` as an input. @tylerflex how would that work with enum?